doc/src/conf.py

```diff
@@ -372,7 +372,13 @@
 \usepackage{letltxmacro}
 \LetLtxMacro\OldLaTeX\LaTeX
 \AtBeginDocument{\DeclareRobustCommand{\LaTeX}{\text{\OldLaTeX}}}
-'''
+\let\OldUnderscore\_
```

Comment:
`\_` in LaTeX is a "robust" macro.  For replacing it I will use the more modern `\protected\def` mechanism.